base/v0_5_exp/schema.go

```diff
@@ -168,20 +168,20 @@ type PasswdGroup struct {
 }
 
 type PasswdUser struct {
-	Gecos             *string            `yaml:"gecos"`
-	Groups            []Group            `yaml:"groups"`
-	HomeDir           *string            `yaml:"home_dir"`
-	Name              string             `yaml:"name"`
-	NoCreateHome      *bool              `yaml:"no_create_home"`
-	NoLogInit         *bool              `yaml:"no_log_init"`
-	NoUserGroup       *bool              `yaml:"no_user_group"`
-	PasswordHash      *string            `yaml:"password_hash"`
-	PrimaryGroup      *string            `yaml:"primary_group"`
-	ShouldExist       *bool              `yaml:"should_exist"`
-	SSHAuthorizedKeys []SSHAuthorizedKey `yaml:"ssh_authorized_keys"`
-	Shell             *string            `yaml:"shell"`
-	System            *bool              `yaml:"system"`
-	UID               *int               `yaml:"uid"`
+	Gecos             *string                    `yaml:"gecos"`
+	Groups            []Group                    `yaml:"groups"`
+	HomeDir           *string                    `yaml:"home_dir"`
+	Name              string                     `yaml:"name"`
+	NoCreateHome      *bool                      `yaml:"no_create_home"`
+	NoLogInit         *bool                      `yaml:"no_log_init"`
+	NoUserGroup       *bool                      `yaml:"no_user_group"`
+	PasswordHash      *string                    `yaml:"password_hash"`
+	PrimaryGroup      *string                    `yaml:"primary_group"`
+	ShouldExist       *bool                      `yaml:"should_exist"`
+	SSHAuthorizedKeys []SSHAuthorizedKeyResource `yaml:"ssh_authorized_keys"`
```

Comment:
I may have misunderstood something, but I think this would be a breaking change for `ssh_authorized_keys`.
I'd defer to @bgilbert as I'm not sure what's the versioning policy for this kind of changes to Butane schemas.

Reply:
Yeah, it's a breaking change and I even put it into the upgrade guides. I am not sure a non-breaking change is feasible or maintainable. Given that the actual change is effectively "prefix your existing keys with `inline: `" the migration should be straightforward.
That said: I'd be happy to work something out with you and improve upon this first attempt :)